### PR TITLE
Add upload API

### PR DESCRIPTION
**Pull request by rofirrim**
First commit to implement the upload API.

### CODE REVIEW

File: app/models/plog.go [hunk 1]
```diff
@@ -14,3 +14,12 @@ type Plog struct {
...
 	return plog.Text
 }
+
+// Used for uploading
+type PlogData struct {
+	Text         string
+	Protagonista int
+	Autor        int
+	Titol        string
+	Data         int64 // Unix epoch
```
Comment (sp-sergi-velez) [inline app/models/plog.go]: `Data` is misleading. Can we use `Time` or `Timestamp` instead?

Comment (risdevs) [inline app/models/plog.go:24]: `Data` is misleading. Can we use `Time` or `Timestamp` instead?

**Review by risdevs — CHANGES_REQUESTED** [review]

File: app/models/plog.go (hunk 1 above)
Comment (rofirrim) [reply app/models/plog.go:24]: Well these fields are in Catalan so it would look a bit off.

**Review by risdevs — APPROVED** [review]

File: app/controllers/app.go [hunk 2]
```diff
@@ -276,3 +276,45 @@ func (c App) SearchJSON(page int) revel.Result {
...
+	}
+
+	plogData := uploadJSON.Upload
+
+	// Check input
+	_, err = GetUser(plogData.Protagonista)
+	// FIXME: Refactor if we need to add more checks like the ones below.
+	if err != nil {
+		revel.ERROR.Println("Error retrieving Protagonista", err)
+		c.Response.Status = http.StatusBadRequest
```
Comment (brafales) [inline app/controllers/app.go:300]: I thought we decided a 400 was not in order here

Comment (rofirrim) [reply app/controllers/app.go:300]: Apparently I didn't understand what the consensus was.

Comment (darmng) [reply app/controllers/app.go:300]: Guys, please keep politics out of this.

**Pull request merged**